spec/Section 3 -- Type System.md

```diff
@@ -1660,6 +1660,12 @@ fragment SomeFragment on SomeType {
 }
 ```
 
+Directive may be defined as repeatable per location with the `repeatable` keyword:
+
+```graphql example
+directive @example repeatable on OBJECT | INTERFACE
+```
+
```

Comment:
Would be great to include a more complex example that illustrates how `repeatable` is useful or at least add a note that directive order is significant and you can use it to create chains of directives.

Reply:
I extended the example https://github.com/facebook/graphql/commit/218e0da6d614bc5be33bc4a6378618ad4a59ea51 and the description https://github.com/facebook/graphql/commit/477919e3251c3b7852a2ec21990bbdf881ff1553.

I struggled with it a bit since I find it hard to strike the right balance between completeness of the example including its description and succinct focused content required from a perspective of the specification. For this reason I also hesitated mentioning the order significance since it has its own dedication section with a separate example.

WDYT?